data/net/minecraft/server/players/GameProfileCache.mapping

```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/server/players/GameProfileCache
 		COMMENT Add an entry to this cache
 		ARG 1 gameProfile
 	METHOD get (Ljava/lang/String;)Ljava/util/Optional;
-		COMMENT Get a player's GameProfile given their username. Mojang's server's will be contacted if the entry is not cached locally.
+		COMMENT Get a player's GameProfile given their username. Mojangs server's will be contacted if the entry is not cached locally.
```

Comment:
In this case I think the original is wrong but it should be changed the other way around: `Mojang's servers`.